src/project-2.js

```diff
@@ -11,15 +18,29 @@ const greeting = (language) => {
   // language: 'English' -> 'Hello!'
   // language: 'Spanish' -> 'Hola!'
   // if language is undefined return 'Hello!'
+  if (language === 'German') {
```

Comment:
If your base case matches one of your cases, you can just drop the else if (language === 'English').
I'd also suggest using switch cases to reduce code size.